packages/Menu/src/MenuList.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { emotion } from '@leafygreen-ui/lib';
+import { colors } from '@leafygreen-ui/theme';
+import { cx } from 'emotion';
+
+const { css } = emotion;
+
+const borderStyle = css`
+  border-bottom: 1px solid ${colors.gray[7]};
+`;
+interface Props {
+  children: React.ReactNode;
+  className?: string;
+}
+
+function MenuList({ children, className }: Props) {
```

Comment:
`export default function...` here. You can still set `displayName` and such afterwards.

Reply:
👍